doc/modules/impute.rst

```diff
@@ -121,6 +121,18 @@ Both :class:`SimpleImputer` and :class:`IterativeImputer` can be used in a Pipel
 as a way to build a composite estimator that supports imputation.
 See :ref:`sphx_glr_auto_examples_plot_missing_values.py`.
 
+Flexibility of IterativeImputer
+===============================
```

Comment:
I think this should be a lower heading level. Then again, perhaps that is true of multiple imputation below too.

Reply:
Sorry, how do I do make this a lower heading level?

Reply:
Underline with something else, e.g. `-----`. Restructured Text follows the order of different heading levels within a document to determine hierarchy.

Reply:
Thanks.